include/pingcap/kv/RegionCache.h

```diff
@@ -36,7 +36,7 @@ struct RegionVerID
     RegionVerID() = default;
     RegionVerID(uint64_t id_, uint64_t conf_ver, uint64_t ver_) : id(id_), confVer(conf_ver), ver(ver_) {}
 
-    bool operator==(const RegionVerID & rhs) const { return id == rhs.id && confVer == rhs.confVer && ver == rhs.ver; }
+    bool operator==(const RegionVerID & rhs) const { return id == rhs.id; }
```

Comment:
why only compare id?

Reply:
Because TiDB client only compares id ...

Reply:
Indeed, tidb use `getRegionByIDFromCache` to locate id and drop region by region_id and version. Only considering id will cause unprecise region cache drop.
However, whether only comparing with id doesn't affect the correctness, and I wish the code cleaner and refine it in future.

Reply:
GetRPCContext needs to match region_id+version. If not, it is possible that the request is composed to send to old region (with broader range), but the context attached is the new region (cache could be updated by other txn). On tikv-server side, it verifies the request by checking version (can pass because context is new), then it execute the request but mutations in the request may exceeds region range.

Reply:
fixed